org.freedesktop.Sdk.Extension.dotnet.yaml

```diff
@@ -0,0 +1,76 @@
+app-id: org.freedesktop.Sdk.Extension.dotnet
```

Comment:
Please format the file according to our style guide: https://github.com/flathub/flathub/wiki/YAML-Style-Guide